src/formatters/codesearch.js

```diff
@@ -37,9 +37,10 @@ const sourcegraphMap = {
 
   lockwise_ios: "lockwise-ios",
 
-  mach: "mach",
+  mach: "gecko-dev",
 
-  mozphab: "mozphab",
+  // https://github.com/mozilla-conduit/review
+  mozphab: "review",
```

Comment:
Hmm, I don't think this one will actually work. I think the owner of mozilla-conduit needs to add their organization to sourcegraph. @zzzeid are you interested in doing that? In addition to making it easier to search for glean metric definitions in mozphab, sourcegraph seems like a generally useful tool.

https://docs.sourcegraph.com/admin/external_service/github

You can see how it works right now for mozregression: https://dictionary.protosaur.dev/apps/mozregression/metrics/usage_app (using the code search tab)

```suggestion
```

Reply:
Hmm I assumed that Sourcegraph indexes all public repositories by default, which `mozilla-conduit/review` is, but I might be wrong. I just tested and it looks like we can search code in mozilla-conduit/review: https://sourcegraph.com/search?q=repo:%5Egithub%5C.com%5C/%5BMm%5Dozilla%28.*%29%5C/review%24+test&patternType=regexp

Reply:
Hmm, interesting! That didn't seem to work the last time I tested it. I'll send in another PR to re-enable it if I can get it working locally.